src/actions/AuthActions.js

```diff
@@ -94,6 +93,24 @@ export const createAccount = ({
 	}
 
 };
+/**
+ * @method isAllWIFsAdded
+ * @param {Object} account
+ * @param {String} password
+ * @return {Boolean}
+ */
+const isAllWIFsAdded = async (account, password) => {
+	const userStorage = Services.getUserStorage();
+	const userWIFKeys = await userStorage.getAllWIFKeysForAccount(account.id, { password });
+	const userPublicKeys = account.active.key_auths;
+	const publicEchorandKey = account.echorand_key;
+	const isPrivateEchorandAdd = userWIFKeys.find((key) => key.wif === publicEchorandKey);
+	if ((userPublicKeys.length > userWIFKeys.length) || !isPrivateEchorandAdd) {
```

Comment:
would be better to `return (userPublicKeys.length === userWIFKeys.length && isPrivateEchorandAdd);`